public/javascripts/averages.js

```diff
@@ -1,69 +0,0 @@
-// Establish empty object with the keys that we're going to use to calculate averages.
-
-// Set the key array length to 0, so we can increment it later
-
-// Create a function that will calculate the averages of a set of reviews we give it. Within this function we will loop through the keys (overallScore, features etc) of the object we established above.
-
-// We loop through each review using the first key (overallScore), looking for the value. We use filter and map to create an array of those values. Eg. Overall Scores = [3,4,3]. If for some reason, there are keys that have not been scored yet, we will skip to the next key from our averages object.
-
-// Then, we take the array we created in the last step and run a reduce on it to get the sum, and then the average. We then insert the average of that key (overallScore) into the averages object we set at the start of the function.
-
-// Once we have looped through all of the keys in averages, we will have an object that contains their average score for each of the keys present.
-
-// The function ultimately returns the average of all metrics, across all the reviews left for a company, so will be our headline number.
```

Comment:
When you're leaving multi-line comments, it's standard practice to use the below as it's much easier to read:

```
/**
 * Establish empty object with the keys that we're going to use to calculate averages.
 * Set the key array length to 0, so we can increment it later
 */